addons/web/static/src/views/fields/percent_pie/percent_pie_field.xml

```diff
@@ -10,7 +10,7 @@
         />
 
         <div class="o_pie_info">
-            <span class="o_pie_value" t-esc="props.record.data[props.name] + '%'"/>
+            <span class="o_pie_value" t-esc="+(props.record.data[props.name].toFixed(2)) + '%'"/>
```

Comment:
Hello @amdi-odoo ,
LGTM, but why the `+` sign? =)
Cheers!

Note: If I get it correctly, toFixed is not a perfect solution for rounding either, but it should be more than enough since called after the rounding in the python methods. :) I do not think we should dig deeper as the issue should almost never occur :p

Reply:
Hi @Odoonan :slightly_smiling_face: 
`toFixed` is returning a string rounded to 2 decimals and the unary `+` is converting it back to a Number.
The `+` is quite useful when the value is an integer as it will automatically remove the unused `0` decimals.
Instead of displaying `100.00`, using the `+` will automatically display `100` instead.
I did that to keep the same display as the current master runbot:
![image](https://github.com/odoo/odoo/assets/107679666/14cc1c25-d56e-4480-9308-e76d36917f65)

Reply:
Shouldn't we move it at JS level so that it is computed in the component ? Normally we avoid doing convert / data manipulation in templates, who should be mainly about formatting and displaying. Here we stand at the border, I feel like it should be done in JS, what do you think ?